openshift/delete_namespace.go

```diff
@@ -0,0 +1,25 @@
+package openshift
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/fabric8-services/fabric8-tenant/configuration"
+	"github.com/pkg/errors"
+)
+
+// DeleteNamespace deletes a namespace on the target Openshift cluster
+func DeleteNamespace(name string, clusterURL string, token string, clientOptions ...configuration.HTTPClientOption) error {
+	_, err := executeRequest(context.Background(),
+		request{
+			method:      "DELETE",
+			url:         fmt.Sprintf("%s/oapi/v1/projects/%s", strings.TrimSuffix(clusterURL, "/"), name),
```

Comment:
why can't we use the client APIs to do this? https://github.com/kubernetes/client-go or if it is a OpenShift specific operation then we can also use https://github.com/openshift/client-go

Reply:
I'm fine with that. @aslakknutsen, what do you think about it ?

Reply:
We could, but Openshift go client has historically been a beast and we only need like 2 simple REST calls

Reply:
ah ok, maybe we would wanna consider moving all our api calls to client-go of openshift!